nipype/interfaces/tests/test_nilearn.py

```diff
@@ -0,0 +1,199 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+import unittest
+import os
+import tempfile
+import shutil
+
+import numpy as np
+
+from ...testing import (assert_equal, utils, assert_almost_equal, raises,
+                        skipif)
+from .. import nilearn as iface
+
+no_nilearn = True
+try:
+    __import__('nilearn')
```

Comment:
AFAIK, `import nilearn` would actually perform the import, while `__import__('nilearn')` only checks that `nilearn` is importable. Since `nilearn` wasn't used at all in this file, that seemed more appropriate. I don't feel strongly about it though.